test/integration/controllers/user.spec.js

```diff
@@ -94,6 +96,28 @@ describe('User controller', () => {
       expectError(404, USER_NOT_FOUND, response)
     })
   })
+  
+  describe(`UPDATE ${endpointUrl}update-my-profile`, () => {
+    const requestUrl = endpointUrl + 'update-my-profile'
+    const mockedFirstName = 'Albus Dumbledore'

```

Comment:
maybe it wil be better to be like mockedName: {firstName: 'Albus', lastName: 'Dumbledore'} ?

Reply:
yeah, u're right